# Intrinsics information for fiducials

# Fiducial point positions in the fiducial frame of reference
geometry_msgs/Point[] points
